Tools/CMake/AMReXThirdPartyLibraries.cmake

```diff
@@ -79,9 +79,10 @@ endif ()
 #
 if (AMReX_SUNDIALS)
     if (SUNDIALS_FOUND)
-        message(STATUS "SUNDIALS_FOUND is true, assuming nvecserial or gpu-specific vector found for version 5.7.0")
+        message(STATUS "SUNDIALS_FOUND is true, assuming nvecserial or gpu-specific vector found for version 6.0.0 or higher")
```

Comment:
Does sundials provide a variable/components that could be checked here? We could also throw an error by checking `SUNDIALS_VERSION` (or equivalent) here to enforce `6.0.0` or higher?

Reply:
In theory, when sundials is installed as a library for the versions of interest a SUNDIALSConfig.cmake file is available. Part of the explicit check on SUNDIALS_FOUND was intended as a workaround for super builds containing both sundials and amrex, where sundials may not be installed to the filesystem.
https://sundials.readthedocs.io/en/latest/Install_link.html?highlight=sundialsconfig#using-sundials-as-a-third-party-library-in-other-cmake-projects

I would prefer to defer using the config file until the next minor Sundials release, as the current config file may depend non-trivially on the new sundials profiling capability. @gardner48 @balos1 thoughts?